serviceUsage.html

```diff
@@ -272,6 +272,27 @@ <h2>
             </tr>
         </table>
 
+        <h5>generateAndPublish endpoint is provided with two options for Lookups in swagger page:</h5>
```

Comment:
This options are to be provided in the parameters above, this explanation can be kept, but the parameters needs to be provided in the usage part.